src/Endpoint/StatusEndpoint.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenEuropa\CdtClient\Endpoint;
+
+use OpenEuropa\CdtClient\Contract\TokenAwareInterface;
+use OpenEuropa\CdtClient\Model\Response\Translation;
+use OpenEuropa\CdtClient\Traits\TokenAwareTrait;
+use OpenEuropa\CdtClient\Traits\ValidationAwareTrait;
+
+/**
+ * Class StatusEndpoint
+ *
+ * Defines how the client should handle requests to the "requests/:requestyear/:requestnumber" space of the API.
+ * Implements the TokenAwareInterface to handle authentication tokens for secure communication.
+ *
+ * @see EndpointBase
+ * @see TokenAwareInterface
+ */
+class StatusEndpoint extends EndpointBase implements TokenAwareInterface
+{
+    use TokenAwareTrait;
+    use ValidationAwareTrait;
+
+    protected string $permanentId;
+
+    public function getPermanentId(): string
+    {
+        return $this->permanentId;
+    }
+
+    public function setPermanentId(string $permanentId): self
+    {
+        if (!preg_match('/^\d{4}\/[^\/]+$/', $permanentId)) {
```

Comment:
I've added the validation to avoid the scenario where someone tries to use only the ID, without the year.